app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_secure_password
+  validates :username, presence: true, uniqueness: true
+  validates :password, presence: true, length: { minimum: 6 }
```

Comment:
As you're using `password_confirmation`, should we add `confirmation: true` here?
https://guides.rubyonrails.org/active_record_validations.html#confirmation

Reply:
`confirmation: true` is not necessary because I'm using `has_secure_password`.
```Confirmation of password (using a XXX_confirmation attribute)``` is added automatically.
ref : https://api.rubyonrails.org/classes/ActiveModel/SecurePassword/ClassMethods.html

Reply:
Thank you for the information, I rarely use `has_secure_password`